Common/Data/Custom/Quiver/QuiverWallStreetBets.cs

```diff
@@ -0,0 +1,133 @@
+using System.Linq;
+using Newtonsoft.Json;
+using System;
+using System.IO;
+using NodaTime;
+using ProtoBuf;
+using static QuantConnect.StringExtensions;
+
+namespace QuantConnect.Data.Custom.Quiver
+{
+    /// <summary>
+    /// Mentions of the given company's ticker in the WallStreetBets daily discussion thread
+    /// </summary>
+    [ProtoContract(SkipConstructor = true)]
+    public class QuiverWallStreetBets : BaseData
+    {
+        /// <summary>
+        /// Date of the daily discussion thread
+        /// </summary>
+        [ProtoMember(10)]
+        [JsonProperty(PropertyName = "Date")]
+        [JsonConverter(typeof(DateTimeJsonConverter), "yyyy-MM-dd")]
+        public DateTime Date { get; set; }
+
+        /// <summary>
+        /// The number of mentions on the given date
+        /// </summary>
+        [ProtoMember(11)]
+        [JsonProperty(PropertyName = "Count")]
+        public int Mentions { get; set; }
+
+        /// <summary>
+        /// Required for successful Json.NET deserialization
+        /// </summary>
+        public QuiverWallStreetBets()
+        {
+        }
+
+        /// <summary>
+        /// Creates a new instance of QuiverWallStreetBets from a CSV line
+        /// </summary>
+        /// <param name="csvLine">CSV line</param>
+        public QuiverWallStreetBets(string csvLine)
+        {
+            // Date[0], Mentions[1]
+            var csv = csvLine.Split(',');
+            Date = Parse.DateTimeExact(csv[0], "yyyyMMdd");
+            Mentions = Parse.Int(csv[1]);
+            Time = Date;
+        }
+
+        /// <summary>
+        /// Return the Subscription Data Source gained from the URL
+        /// </summary>
+        /// <param name="config">Configuration object</param>
+        /// <param name="date">Date of this source file</param>
+        /// <param name="isLiveMode">true if we're in live mode, false for backtesting mode</param>
+        /// <returns>Subscription Data Source.</returns>
+        public override SubscriptionDataSource GetSource(SubscriptionDataConfig config, DateTime date, bool isLiveMode)
+        {
+            var source = Path.Combine(
+                Globals.DataFolder,
+                "alternative",
+                "quiver",
+                "wallstreetbets",
+                $"{config.Symbol.Value.ToLowerInvariant()}.csv"
+            );
+            return new SubscriptionDataSource(source, SubscriptionTransportMedium.LocalFile, FileFormat.Csv);
+        }
+
+        /// <summary>
+        /// Reader converts each line of the data source into BaseData objects.
+        /// </summary>
+        /// <param name="config">Subscription data config setup object</param>
+        /// <param name="line">Content of the source document</param>
+        /// <param name="date">Date of the requested data</param>
+        /// <param name="isLiveMode">true if we're in live mode, false for backtesting mode</param>
+        /// <returns>
+        /// Quiver Twitter object
```

Comment:
nit - Same comment about `Quiver Twitter object` . Think this same comment applies for the rest too